src/css/index.css

```
@@ -13,6 +14,7 @@
 @import url('./modules/theme.css');
 @import url('./modules/text.css');
 @import url('./modules/button.css');
+@import url('./modules/popup.css');
```

Stylelint (17.14.0)

[error] Expected "url('./modules/popup.css')" to be "'./modules/popup.css'" (import-notation)

(import-notation)